packages/esm-hiv-app/src/ugandaemr-config.ts

```diff
@@ -0,0 +1,20 @@
+import ugandaEmrLogo from './images/ugandaemr_+_logo.svg';
+import ugandaEmrBannerLogo from './images/ugandaemr_+_logo_white.svg';
+
+export default {
```

Comment:
These configurations already exist in the core package and should be removed from here